common/ux-alignment.cpp

```diff
@@ -49,8 +50,7 @@ bool is_gui_aligned(GLFWwindow *win)
 
         glfwSwapBuffers(win);
 
-        ImGui_ImplGlfw_NewFrame();
-        ImGui::NewFrame();
+        ImGui::PushNewFrame();
```

Comment:
Why do we reuse the ImGui namespace and not place all of our special functions under RsImGui?

Reply:
you are right, at first it was mostly overloading functions and in order not to change the code I used the ImGui namespace but now it has helper functions, so it makes more sense to change it.